basic_pipeline_extension/01_Introduction.md

```diff
@@ -0,0 +1,4 @@
+# Introduction
+
+In this section, we gathered some topics which should give you better insight into Membrane. We will base on the [basic pipeline tutorial](../basic_pipeline/01_Introduction.md), so make sure you have gone through it first.
+The chapters are loosely related, so feel free to read them in any order you want.
```

Comment:
How about listing the chapters of the tutorial here, and shortly describing (just like a single sentence) each of them?

Reply:
This is a good idea!